MusicHub/users/models.py

```diff
@@ -12,9 +11,16 @@
     PermissionsMixin,
 )
 
+from authemail.models import EmailAbstractUser, EmailUserManager
+
 
 class CustomManager(BaseUserManager):
     def create_user(self, email, password, first_name, last_name, **kwargs):
+        """
+        Creates and saves a User with a given email and password.
+        """
+        if not email:
+            raise ValueError("User must provide an email address")
```

Comment:
I think that Django checks that automatically, so we don't have to check if email is present